util-cache/src/main/scala/sbt/util/ActionCacheStore.scala

```diff
@@ -263,24 +266,43 @@ class DiskActionCacheStore(base: Path) extends AbstractActionCacheStore:
       if ref.id.startsWith("${OUT}/") then ref.id.drop(7)
       else ref.id
     val d = Digest(ref)
-    def symlinkAndNotify(outPath: Path): Path =
+    def copyFile(outPath: Path): Path =
+      Files.copy(
+        casFile,
+        outPath,
+        StandardCopyOption.COPY_ATTRIBUTES,
+        StandardCopyOption.REPLACE_EXISTING,
+      )
+    // See https://github.com/sbt/sbt/issues/7656
+    // On Windows, the program has be running under the Administrator privileges or the
+    // user enable Developer Mode on Windows 10+ to create symbolic links.
+    def writeFileAndNotify(outPath: Path): Path =
       Files.createDirectories(outPath.getParent())
       val result = Retry:
         if Files.exists(outPath) then IO.delete(outPath.toFile())
-        Files.createSymbolicLink(outPath, casFile)
+        if symlinkSupported.get() then
```

Comment:
We had briefly discussed updating this to prevent repeat attempts to create links. Is that still the intent here?

Reply:
Yea good catch. I didn't update the atomic boolean.

Reply:
Addressed in aa0ea6ee9f26abc0c2467945051a14d10f44abe7